Oqtane.Client/Modules/Controls/Label.razor

```diff
@@ -40,5 +46,19 @@ else
         }
 
         _openLabel += ">";
+
+        if (!string.IsNullOrEmpty(ResourceKey))
+        {
+            if (ModuleState?.ModuleType != null)
+            {
+                var moduleType = Type.GetType(ModuleState.ModuleType);
+                var localizerTypeName = $"Microsoft.Extensions.Localization.IStringLocalizer`1[[{moduleType.AssemblyQualifiedName}]], Microsoft.Extensions.Localization.Abstractions";
+                var localizerType = Type.GetType(localizerTypeName);
+                var localizer = (IStringLocalizer)HttpContextAccessor.HttpContext.RequestServices.GetService(localizerType);
+                
+                ChildContent = @<text>@localizer[$"{ResourceKey}.Text"]</text>;
```

Comment:
@sbwalker I don't see any text property for a label component, but I find this line work to set the label text, please let me know your feedback about this line

Reply:
this looks good! what would the RESX file look like to define the localized values?

Reply:
Basically as:

Name.Text => Nom

Name.HelpText => Entrez Le num do site

Where `Name` is the resource name provided in `ResourceKey` property